src/_main_/utils/emailer/send_email.py

```diff
@@ -38,6 +37,7 @@ def send_massenergize_email(subject, msg, to):
     text=msg, 
   )
   response = pystmark.send(message, api_key=EMAIL_POSTMARK_SERVER_TOKEN)
+  response.raise_for_status()
```

Comment:
this should be in send_massenergize_email_with_attachments which is used to send the guest email using a template